Python/libraries/recognizers-number/recognizers_number/number/parsers.py

```diff
@@ -139,7 +139,7 @@ def __init__(self, config: NumberParserConfiguration):
         self.round_number_set: List[str] = list(
             self.config.round_number_map.keys())
         self.is_non_standard_separator_variant = self.config.culture_info.code in \
-            self.config.non_standard_separator_variants
+                                                 self.config.non_standard_separator_variants
```

Comment:
Nitpick: Don't change indentation, unless it currently violate the default Python validation rules.